src/content/platform-integration/web/embedding-flutter-web.md

```diff
@@ -1,20 +1,25 @@
 ---
-title: Embedding Flutter on the web
-short-title: Embedding Flutter web
+title: Adding Flutter to any web application
+short-title: Add Flutter to any web app
```

Comment:
Please, @kevmoo, take a look at the changes to the title and left navbar; here's what they look like with this PR:

<img width="940" alt="Screenshot 2024-09-12 at 6 31 32 PM" src="https://github.com/user-attachments/assets/c6395c99-69ca-4597-9106-0f9a4aeabcc3">

They're the smartest that I can come up with 😛